plugins/lime-plugin-metrics/src/metricsApi.js

```diff
@@ -1,11 +1,14 @@
-export const getMetrics = (api, sid,params) => api.call(sid, 'lime-metrics', 'get_metrics', params)
-	.map(result => Object.assign({}, result, { target: params.target })
-	);
+import { map } from 'rxjs/operators';
+
+export const getMetrics = (api, sid,params) => api.call(sid, 'lime-metrics', 'get_metrics', params).pipe(
+	map(result => Object.assign({}, result, { target: params.target }))
+);
 
 export const getAllMetrics = (api, sid, params) => params.targets.map(x => getMetrics(api,sid, { target: x }));
 
-export const getGateway = (api, sid) => api.call(sid, 'lime-metrics', 'get_gateway', {})
-	.map(result => result);
+export const getGateway = (api, sid) => api.call(sid, 'lime-metrics', 'get_gateway', {}).pipe(
+	map(result => result)
```

Comment:
Is this map call necessary here? I'm not familiar with the API.

Reply:
Nice catch!

Reply:
fixed on 49e4457